### PR TITLE
adds tracing for markup strings

### PR DESCRIPTION
**Pull request by christophkloeffel**
adds tracing for chapter 19

### CODE REVIEW

File: tests-system/lint-ug-examples/output [hunk 1]
```diff
@@ -17,8 +17,8 @@ len(x) >= 3 implies x[3] > 0, "too small"
     ^ lint-ug-examples/div_by_zero.rsl:12: issue: divisor could be 0 [vcg-div-by-zero]
     | example record_type triggering error:
     |   T bad_potato {
-    |     x = 3
```
Comment (phiwuu) [inline tests-system/lint-ug-examples/output:20]: Please revert

File: trlc/ast.py [hunk 2]
```diff
@@ -1074,6 +1074,7 @@ def evaluate(self, mh, context):
         return Value(self.location, self, self.typ)
 
     def resolve_references(self, mh):
+        # lobster-trace: LRM.Markup_String_Late_Reference_Resolution
```
Comment (phiwuu) [inline trlc/ast.py:1077]: move to https://github.com/bmw-software-engineering/trlc/blob/92947777b0d034c37b2db1fb82502c420ca50326/trlc/trlc.py#L507 instead (function `resolve_record_references` in `trlc.py`)

File: trlc/parser.py [hunk 3]
```diff
@@ -216,6 +220,7 @@ def match_kw(self, value):
 
 
 class Markup_Parser(Parser_Base):
+
```
Comment (phiwuu) [inline trlc/parser.py:223]: remove empty line

File: trlc/trlc.py [hunk 4]
```diff
@@ -589,6 +589,7 @@ def process(self):
         # Resolve record reference names and do the missing semantic
         # analysis.
         # lobster-trace: LRM.File_Parsing_References
+        # lobster-trace: LRM.Markup_String_Late_Reference_Resolution
```
Comment (phiwuu) [inline trlc/trlc.py:592]: move into function `resolve_record_references`, see comment above

File: trlc/parser.py [hunk 5]
```diff
@@ -226,6 +231,7 @@ def __init__(self, parent, literal):
         self.references = literal.references
 
     def parse_all_references(self):
+        # lobster-trace: LRM.Markup_String_Late_Reference_Resolution
```
Comment (phiwuu) [inline trlc/parser.py:234]: delete here, see other comments

**Review by phiwuu — CHANGES_REQUESTED** [review]

File: tests-system/lint-ug-examples/output (hunk 1 above)
Comment (christophkloeffel) [reply tests-system/lint-ug-examples/output:20]: fixed with options file like florian mentioned #78

**Review by phiwuu — DISMISSED** [review]

**Review by phiwuu — APPROVED** [review]

**Pull request merged**